include/cocaine/rpc/asio/encoder.hpp

```diff
@@ -111,7 +112,23 @@ struct encoded:
         typedef typename event_traits<Event>::argument_type argument_type;
 
         type_traits<argument_type>::pack(packer, std::forward<Args>(args)...);
+        packer.pack_array(3);
+
+        auto trace_id = 0; //tracer::current_span()->get_trace_id();
+        header_t::str trace_id_data{reinterpret_cast<const char*>(&trace_id), 8};
```

Comment:
headers are binary strings. here we pack uint64_t to binary string of 8 bytes. I've thought about adding a convinient method, but it seems it will be even less obvious that it packs numbers via binary representation.